### DISCUSSION TITLE
Custom types are not applied to the @OneToMany decorator

### DISCUSSION THREAD

**Discussion by james-zent** [mikro-orm/mikro-orm#4347] [Q&A]
**Describe the bug**
It seems that the log is not taken, so it does not apply.
I saw that it was applied to the @Property decorator, but it doesn't seem to work. The top-level parent of the @OneToMany decorator is known as @Property.

**Stack trace**

**To Reproduce**
```typescript
// custom-types.ts
export class CustomTypes extends Type<any, any> {
  convertToDatabaseValue(
      value: any,
      platform: Platform,
      context?: TransformContext | boolean,
    ): any {
      console.log(value);
      return super.convertToDatabaseValue(value, platform, context);
    }
  
    convertToJSValue(value: any, platform: Platform): any {
      console.log(value);
      return super.convertToJSValue(value, platform);
    }
}

// member.ts
@Entity()
export class Member {
   ...
  @OneToMany(Board, 'member', {customTypes: new CustomTypes()})
  boards = new Collection<Board>(this);
}

// board.ts
@Entity()
export class Board {
  ...
  @ManyToOne({entity: Member, joinColumns:'member_id'})
  member: Member;
}
```

**Expected behavior**
console.log should be output.

**Additional context**

**Versions**

| Dependency | Version |
| - | - |
| node | 16.17.1 |
| typescript | 4.3.5 |
| mikro-orm | 5.7.5 |
| your-driver | postgresql |

**Comment by B4nan** [comment]
What do you expect from this? Custom types are only valid on scalar properties, not on relations. You can use them to map the relation's primary key, not to map the relation itself.
